_includes/script.html

```diff
@@ -1,8 +1,11 @@
 <script type="text/javascript">
     $('h1, h2, h3, h4, h5, h6').filter('[id]').each(function () {
-        var el = document.createElement('a')
-        el.className = 'anchor'
+        var el = document.createElement('a');
+        el.className = 'anchor';
         el.href = '#' + this.id;
-        this.prepend(el)
+        this.prepend(el);
     });
+    $('pre code').each(function () {
+        this.innerHTML = this.innerHTML.replace(/\u2502/g, '<u>\u2502</u>');
```

Comment:
I'm not sure I understand why you're using an `<u>` element instead of a `<span>`. The latter makes more semantic sense IIUC, and using `<u>` forces you to override the underlining with `text-decoration: none;` in the CSS. What's the rationale? Is it just because it's shorter?

Reply:
Because it's shorter, but also because `<span>` is already used within `<code>` for syntax highlighting. `<u>` has no _semantic_ meaning in HTML5, so it shouldn't be a problem.